content/source/docs/enterprise/api/team-access.html.md

```diff
@@ -160,13 +163,9 @@ $ curl \
 
 ## Show Team Access to a Workspace
 
-| Method | Path           |
-| :----- | :------------- |
-| GET | /team-workspaces/:id |
-
-### Parameters
+`GET /team-workspaces/:id``
 
-- `id` (`string: <required>`) - The ID of the team/workspace relationship.
+- `:id` - The ID of the team/workspace relationship. Obtain this from the "list team access" action described above.
```

Comment:
In Markdown (or at least GitHub flavoured) you can reference sections on the same page using `[text](#id)`.